doc/thing/thing-create-app.rst

```diff
@@ -130,9 +130,9 @@ On ``loop()`` function you can include your application logic.
 
     ``loop()`` function must NOT be blocking.
 
-On ``setup()`` you can register your sensors/actuators using `knot_proxy_register() <thing-api.html#knot-proxy-register>`_ and configures which events should send data to cloud with `knot_proxy_set_config() <thing-api.html#knot-proxy-set-config>`_.
+On ``setup()`` you can register your sensors/actuators using `knot_data_register() <thing-api.html#knot-data-register>`_ and configures which events should send data to cloud with `knot_data_config() <thing-api.html#knot-data-config>`_.
 
-For each registered sensor/actuator you may want to create a ``changed_cb`` or ``pool_cb`` function, this function are passed as callback on `register <thing-api.html#knot-proxy-register>`_ function.
+For each registered sensor/actuator you may want to create a ``write_cb`` or ``read_cb`` function, this function are passed as callback on `register <thing-api.html#knot-data-register>`_ function.
```

Comment:
Same as before! The changes you made seem good :smile: 

Though, there are a few grammar mistakes that you may want to consider fixing: 

1. `function, this function... `: `this` is a **singular** proximal demonstrative pronoun. I believe the phrase means to refer to both `write_cb` and `read_cb`; therefore, one of the possible corrections would be `function, these functions...`.
2. Also in `function, this functions...`: the comma shouldn't be there ((you can find some rules on comma usage [here](https://www.businessinsider.com/a-guide-to-proper-comma-use-2013-9)). A more suitable alternative would be `function. These functions are then passed as callbacks on...`. If you really must use a comma, then the phrasing should change to something like `function, which will be passed as callbacks on...`

Reply:
Done!